lib/datadog/open_feature/ext.rb

```diff
@@ -3,11 +3,24 @@
 module Datadog
   module OpenFeature
     module Ext
+      # OpenFeature error codes  
       ERROR = 'ERROR'
       INITIALIZING = 'INITIALIZING'
       UNKNOWN_TYPE = 'UNKNOWN_TYPE'
       PROVIDER_FATAL = 'PROVIDER_FATAL'
       PROVIDER_NOT_READY = 'PROVIDER_NOT_READY'
+      TYPE_MISMATCH = 'TYPE_MISMATCH'
+      PARSE_ERROR = 'PARSE_ERROR'
+      FLAG_NOT_FOUND = 'FLAG_NOT_FOUND'
```

Comment:
There are also in

https://github.com/DataDog/libdatadog/blob/3d9e641016f3a6c05bfdb1786c4b1a84342d4bbf/datadog-ffe-ffi/src/assignment.rs#L133-L142